### PR TITLE
[#1973] Remove google web fonts request. Use a list of default fonts until we decide which we want to use

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733-popcorn-maker/tickets/1973-remove-google-web-fonts-request

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/330823292b4e131/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/ffb19dd3c54e68e/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/330823292b4e131/output.txt

Total script time: 0.86 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/ffb19dd3c54e68e/output.txt

Total script time: 0.79 mins

Server is running on http://pr769.butyr.org/templates/basic/
Run the Test Suite: http://pr769.butyr.org/test/

File: src/editor/trackevent-editor.js [hunk 1]
```diff
@@ -291,7 +290,7 @@ define( [ "util/lang", "util/keys", "./base-editor",
           for ( m = 0, fLen = __googleFonts.length; m < fLen; m++ ) {
             font = document.createElement( "option" );
 
-            font.value = font.innerHTML = __googleFonts[ m ].family;
+            font.value = font.innerHTML = __googleFonts[ m ];
```
Comment (jbuck) [inline src/editor/trackevent-editor.js]: `font.value = font.label = ...`

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/abb9b7669f080f3/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/abb9b7669f080f3/output.txt

Total script time: 0.62 mins

Server is running on http://pr769.butyr.org/templates/basic/
Run the Test Suite: http://pr769.butyr.org/test/

**Pull request merged**